### DISCUSSION TITLE
does fastlane match nuke development revoke distribution profile ?

### DISCUSSION THREAD

**Discussion by amazted** [fastlane/fastlane#19780] [Q&A]
Yesterday i run the fastlane  match nuke development to revoke the development cert and provisioning profile associate with development cert. 
But some provisioning file associate with distribution cert become invalid after the command .
![image](https://user-images.githubusercontent.com/8012287/149071987-7b1ff145-01f5-4c78-96ac-63270993621d.png)

**Comment by lucgrabowski** [comment]
Hi, no, it doesn't (https://docs.fastlane.tools/actions/match/#nuke)
Have you changed app identifier by adding or removing capabilities? If you did, that's what invalidated all the profiles for this app identifier.

**Comment by Keon241** [comment]
HELLO,
You can choose to nuke just only a specific environment of certificates and provisioning profiles by providing profile type ( type ). After clearing your account, you'll start from a clean slate, and you can run match to generate your certificates and profiles again

[CUIMS Login](https://www.cuims.net/)
